site/en/guides/pyspinel/requirements.md

```diff
@@ -0,0 +1,53 @@
+# Packet Sniffing Requirements
+
+To use Pyspinel, you'll need to meet the following hardware and software
+requirements.
+
+## Hardware
+
+A host machine to serve as an NCP and to run Pyspinel and Wireshark:
```

Comment:
Hardware and Software sections are inconsistent.  Software is all a list while this is not.  Can you make Hardware a list too to match?  I think we can just move the Note to be a nested bullet (i.e. no longer a note) under the 1 OpenThread device bullet.